superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -744,6 +748,15 @@ export function useDatabaseValidation() {
                         ),
                       };
                     }
+                    if (extra.issue_codes?.length) {
+                      return {
+                        ...obj,
+                        error_type,
+                        description:
+                          message || extra?.issue_codes?.[0]?.message,
```

Comment:
we don't need conditional sign `?` here, because you already checked it in line# 751. 
Just use `extra.issue_codes[0].message`

Btw, are you sure that `issue_code` has always message?

Reply:
nop, and in that case it will fallback to show the json (like the others)